src/fairchem/core/preprocessing/atoms_to_graphs.py

```diff
@@ -163,10 +164,16 @@ def convert(self, atoms: ase.Atoms, sid=None):
         """
 
         # set the atomic numbers, positions, and cell
+        positions = np.array(atoms.get_positions(), copy=True)
+        pbc = np.array(atoms.pbc, copy=True)
+        cell = np.array(atoms.get_cell(complete=True), copy=True)
+        positions = wrap_positions(positions, cell, pbc=pbc, eps=0)
```

Comment:
Is there specific reason for setting `eps=0` vs the default 1e-7?

Reply:
I used `eps=0` following `ase.neighborlist`: https://gitlab.com/ase/ase/-/blob/master/ase/neighborlist.py#L960